### DISCUSSION TITLE
Deployed frontend to Vercel for safe keeping

### DISCUSSION THREAD

**Discussion by EnZon3** [EnZon3/Drop#2] [Announcements]
Drop is basically a skeleton, but sooner or later will have some functionality.

I deployed it to Vercel right now just to get that step out of the way as I develop Drop.

**Comment by boston2029** [comment]
Are you developing locally? How often will you deploy your progress, if you do?

**Reply by EnZon3** [reply]
Yes, I am developing locally. I will deploy my progress whenever I make a feature that I'm working on stable for deployment.
